src/app/home-page/dashboard/dashboard.component.html

```diff
@@ -18,8 +18,10 @@
         <div fxFlex.md="65" fxFlex.gt-md="65" fxFlex.lt-md="100" class="py-4">
           <app-news-box></app-news-box>
         </div>
-        <div fxFlex.md="35" fxFlex.gt-md="35" fxFlex.lt-md="100">
-          <div class="my-3 twitter" #twitter></div>
+        <div class="mt-timeline">
```

Comment:
Lines 21-22 and 24-25 should probably be part of the the app-mastodon-timeline component.